decidim-elections/app/services/decidim/votings/census_vote_flow.rb

```diff
@@ -30,12 +36,20 @@ def voter_data
         }
       end
 
-      def no_access_message
-        I18n.t("vote_flow.datum_not_found", scope: "decidim.votings.census")
-      end
-
-      def login_path(vote_path)
-        EngineRouter.main_proxy(election.component.participatory_space).voting_login_path(election_id: election.id, vote_path: vote_path)
+      def can_vote?(online_vote_path: nil)
```

Comment:
💬  What do you think about calling this `vote_permissions` that would return an `OpenStruct` like
```
{
   allowed?: Boolean,
   error_message: String,
   exit_path: String
}
```